next/frontend/constants/constants.tsx

```diff
@@ -451,7 +451,7 @@ export const serviceCards: ServiceCardBase[] = [
     tag: 'account_section_services.cards.43.tag',
     tagStyle: 'text-main-700 bg-main-100',
     category: [MunicipalServicesCategories.TAXES_CATEGORY],
-    href: ROUTES.MUNICIPAL_SERVICES_TAX,
+    href: ROUTES.MUNICIPAL_SERVICES_FORM('platba-dane-z-nehnutelnosti'),
```

Comment:
`priznanie-k-dani-z-nehnutelnosti`? That is the slug for that form from the former constant.

Reply:
Omg, yes.